### PR TITLE
Establishes COUNTRY_CODE_TO_COUNTRY_NAME and country_name_from_code.

### PR DESCRIPTION
**Pull request by ashercodeorg**
The COUNTRY_CODE_TO_COUNTRY_NAME map was populated using the results of "SELECT code_s, name_s FROM geography_countries;".

This is an initial step towards removing the geography_countries pegasus table. Next steps will replace existing usages of pegasus/src/database.rb:country_name_from_code with lib/country_codes.rb:country_name_from_code and run a migration to delete the geography_countries table.

Related work will also be done to remove the geography_us_states and geography_us_zip_codes pegasus tables. In all cases, it seems unnecessary and inefficient to store these maps as a DB table rather than a ruby hash.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
CC @laurelfan

**Comment by ashercodeorg** [comment]
The test failure (QueueProcessorTest) seems spurious.

**Comment by tanyaparker** [comment]
Wow, I didn't even know about the geography_countries table. Are these mappings used for geo locations like the leaderboard? Because depending on what this is used for, I would have thought that if the code isn't in the hash then we should return some kind of error or bucket with "Other" like it couldn't be categorized. It seems weird for the method to imply it will give a country name but still gives a code.

File: lib/country_codes.rb [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  "YE" => "Yemen",
+  "YT" => "Mayotte",
+  "ZA" => "South Africa",
+  "ZM" => "Zambia",
+  "ZW" => "Zimbabwe",
+}
+
+# Returns the name of the country whose two character country code is code.
+# If code is not a valid two character country code, returns code.
+def country_name_from_code(code)
```
Comment (tanyaparker) [inline lib/country_codes.rb:250]: Can you do this?

```
def country_name_from_code(code)
  return COUNTRY_CODE_TO_COUNTRY_NAME[code.to_s.strip.upcase] || code
end
```

Comment (tanyaparker) [inline lib/country_codes.rb:250]: Since we know that the country name is never going to be nil?

**Comment by ashercodeorg** [comment]
Tracing back, the DB table is used in various places: https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=country_name_from_code&type=Code

As for desired behavior when the code is not found, it is probably worth investigating, though definitely outside the scope of this cleanup.

File: lib/country_codes.rb (hunk 1 above)
Comment (ashercodeorg) [inline lib/country_codes.rb:250]: Nice, I (stupidly) copied the existing implementation. Done (momentarily).

**Comment by tanyaparker** [comment]
Nice, then for the purposes of clean up, lgtm.

**Comment by ashercodeorg** [comment]
Errors seem unrelated (related to NPM and JS).

**Pull request merged**